service/docs/source/geopm_pio_nvml.7.rst

```diff
@@ -0,0 +1,205 @@
+
+
+      *  ``Aggregation``: Average
+      *  ``Domain``: GPU
+      *  ``Format``: Double
+      *  ``Unit``: None
+``NVML::GPU_POWER``:
+    GPU Power usage in watts
+
+
+      *  ``Aggregation``: Sum
+      *  ``Domain``: GPU
+      *  ``Format``: Double
+      *  ``Unit``: Watts
+``NVML::GPU_MEMORY_FREQUENCY_STATUS``
+    GPU memory frequency in Hertz
+
+
+      *  ``Aggregation``: Average
+      *  ``Domain``: GPU
+      *  ``Format``: Double
+      *  ``Unit``: Hertz
+``NVML::GPU_THROTTLE_REASONS``
+    GPU clock throttling reasons.  Refer to NVIDIA NVML documentation for encoding information.
+
+
+      *  ``Aggregation``: Expect_same (note: this is incorrect.  File bug and fix)
+      *  ``Domain``: GPU
+      *  ``Format``: Double
+      *  ``Unit``: None
+``NVML::GPU_TEMPERATURE``
+    GPU Temperature in degrees Celsius
+
+
+      *  ``Aggregation``: Average
+      *  ``Domain``: GPU
+      *  ``Format``: Double
+      *  ``Unit``: Celsius
+``NVML::GPU_ENERGY_CONSUMPTION_TOTAL``
+    GPU energy consumptoin in joules since the driver was loaded
+
+
+      *  ``Aggregation``: Sum
+      *  ``Domain``: GPU
+      *  ``Format``: Double
+      *  ``Unit``: Joules
+``NVML::GPU_PERFORMANCE_STATE``
+    GPU performance state, defined by the NVML API as a value from 0 to 15
+
+
+      *  ``Aggregation``: expect same
+      *  ``Domain``: GPU
+      *  ``Format``: Double
+      *  ``Unit``: None
+``NVML::GPU_PCIE_RX_THROUGHPUT``
+    GPU PCIE receive throughput in Bytes per Second over a 20 millisecond period
+
+
+      *  ``Aggregation``: Sum
+      *  ``Domain``: GPU
+      *  ``Format``: Double
+      *  ``Unit``: Bytes/Second
+``NVML::GPU_PCIE_TX_THROUGHPUT``
+    GPU PCIE transmit throughput in Bytes per Second over a 20 millisecond period
+
+
+      *  ``Aggregation``: Sum
+      *  ``Domain``: GPU
+      *  ``Format``: Double
+      *  ``Unit``: Bytes/Second
+``NVML::GPU_CPU_ACTIVE_AFFINITIZATION``
+    GPU associated with the specified cpu as determined by querying active processes on the GPU.  If no GPUs map to the CPU -1 is returned.  If multiple GPU map to the CPU NAN is returned.
+
+
+      *  ``Aggregation``: Expect_same
+      *  ``Domain``: CPU
+      *  ``Format``: Double
+      *  ``Unit``: None
+``NVML::GPU_MEMORY_UTILIZATION``
+    Fraction of time the GPU memory was accessed in the last set of driver samples
+
+
+      *  ``Aggregation``:
+      *  ``Domain``: GPU
+      *  ``Format``: Double
+      *  ``Unit``: None
+``NVML::GPU_FREQUENCY_MAX_AVAIL``
+    Streaming Multiprocessor maximum frequency in hertz
+
+
+      *  ``Aggregation``: Expect_same
+      *  ``Domain``: GPU
+      *  ``Format``: Double
+      *  ``Unit``: Hertz
+``NVML::GPU_FREQUENCY_MIN_AVAIL``
+    Streaming Multiprocessor minimum frequency in hertz
+
+
+      *  ``Aggregation``: Expect_same
+      *  ``Domain``: GPU
+      *  ``Format``: Double
+      *  ``Unit``: Hertz
+
+Signal Aliases
```

Comment:
This section needs to have its style aligned with other similar sections.  E.g. replace "aliases" with "maps", and put it all on a single line.  This also needs to be done for the L0 page.  We should hash out how we want this to look and then apply it to all the pages.